sound/soc/sof/debug-dsp-ops.c

```diff
@@ -63,6 +63,14 @@ static int sof_dsp_ops_set_power_state(struct snd_sof_dev *sdev, char *state)
 	return 0;
 }
 
+static void sof_dsp_ops_unload_firmware(struct snd_sof_dev *sdev)
+{
+	if (sdev->basefw.fw)
+		snd_sof_fw_unload(sdev);
```

Comment:
Not sure what this does? Is this actually a capability?

Reply:
there's no unloading the firmware from the DSP but I need to reset the sdev->base_fw to be able to start from a clean state for the next iteration

Reply:
a comment would help then, things might be different on other platforms.